ragna/assistants/_anthropic.py

```diff
@@ -93,4 +86,4 @@ class Claude(AnthropicApiAssistant):
         - `httpx_sse`
     """
 
-    _MODEL = "claude-2"
+    _MODEL = "claude-3-opus-20240229"
```

Comment:
This would be `ClaudeOpus`. What about `ClaudeSonnet` and `ClaudeHaiku`? https://docs.anthropic.com/claude/docs/models-overview#model-recommendations